test/playback.test.js

```diff
@@ -31,16 +37,23 @@ QUnit.module('Playback', {
     // videojs.log.level('debug');
     video.style = 'display: none;';
 
+    video.setAttribute('controls', '')
+    video.setAttribute('muted', '')
     video.width = 600;
     video.height = 300;
+    video.defaultPlaybackRate = 16;
```

Comment:
I think it may be useful to still have one simple playback test that runs at a playback rate of 1.